src/controller/artifact/controller.go

```diff
@@ -227,6 +227,7 @@ func (c *controller) ensureArtifact(ctx context.Context, repository, digest stri
 		if !errors.IsConflictErr(err) {
 			return false, nil, err
 		}
+		log.Debugf("failed to create artifact: %v", err)
```

Comment:
The artifact info should be printed, which will be helpful to catch which artifact failed to create due to the conflict.

Reply:
Done PTAL